### DISCUSSION TITLE
Backup question.

### DISCUSSION THREAD

**Discussion by El3ctricFX** [dooit-org/dooit#199] [Q&A] [answered]
is there a quick way to backup the todo list through db files or even just txt files??? thanks

**Comment by kraanzu** [comment, marked as answer]
Yes it should be located in `~/.local/share/dooit`

**Comment by El3ctricFX** [comment]
Appreciate it,

Great Job with the project btw

**Reply by kraanzu** [reply]
Thanks man! I'll be releasing a new version today hopefully. Do check it out :)
